runtime.go

```diff
@@ -593,8 +595,13 @@ func parseOutput(w http.ResponseWriter, stdoutTxt io.Reader, runtime string, wg
 		w.WriteHeader(int(statusCode))
 	}
 
-	// Decode base64 encoded body if isBase64Encoded is true
-	if proxy.IsBase64Encoded {
+	//API Gateway only honors the first Accept media type.
+	acceptMediaType := strings.Split(acceptHeader, ",")[0]
+	contentType := proxy.Headers["Content-Type"]
+	contentMediaType, _, err := mime.ParseMediaType(contentType)
+	acceptMediaTypeMatched := err == nil && acceptMediaType != "" && acceptMediaType == contentMediaType
```

Comment:
The accept header can be blank - you should be able to ignore it if it is

Reply:
Done!